src/odemis/driver/test/smaract_test.py

```diff
@@ -332,6 +333,21 @@ def test_pivot_set(self):
             self.dev.updateMetadata({model.MD_PIVOT_POS: old_pivot})
             self.dev.moveRelSync({"x": 0})
 
+    def test_reference_and_deactivate_move(self):
+        # Set a deactive position and check to be sure that the controller moves to this location
+        # after a reference move
+        # TODO: UIse real physical deactiva position
+        de_pos = {'x': 3.5801e-4, 'y': 0, 'z': 1e-3, 'rx':-1.2e-6, 'rz':-1.453e-6}
```

Comment:
Let's drop rz, to check that it works even if not all axes are present.

Reply:
So... it doesn't work, but this is because of the assert_pos_almost_equal requiring the dict keys to match.